src/main/java/com/redhat/devtools/lsp4ij/installation/ServerInstallerBase.java

```diff
@@ -134,7 +134,9 @@ public void run(@NotNull ProgressIndicator indicator) {
                     install(indicator);
 
                     markAsInstalled(installFuture);
-                } catch (CancellationException | ProcessCanceledException e) {
+                } catch (ProcessCanceledException e) {

```

Comment:
The original form wouldn't compile in 2024.3 when I tested in that version, so I changed it slightly so that it would compile in 2023.3-2024.3.